code/svb-docker/python/svb_web/core/models.py

```diff
@@ -117,16 +117,49 @@ def connect_to_printer(self):
             self.ip_address
         )
 
-    def print_deposit_receipt(self, account: Account) -> None:
+    def _print_header(self):
+        import os
+        import PIL.Image
+        from svb_web.settings import STATIC_ROOT
+        banner_pathname = os.path.join(
+            STATIC_ROOT,
+            "core",
+            "assets",
+            "svb_banner.png"
+        )
+        default_banner_img = PIL.Image.open(banner_pathname)
+        small_banner_img = default_banner_img.resize(
+            [x // 4 for x in default_banner_img.size]

```

Comment:
Why the magic number? Make it a meaningfully-named constant if you have a reason for it